modules/serverpod_auth/serverpod_auth_server/lib/src/endpoints/google_endpoint.dart

```diff
@@ -149,7 +149,8 @@ class GoogleEndpoint extends Endpoint {
       }
 
       var data = jsonDecode(response.body);
-      if (data['iss'] != 'accounts.google.com') {
+      if (data['iss'] != 'accounts.google.com' &&
+          data['iss'] != 'https://accounts.google.com') {
```

Comment:
Suggestion: Change this to an OR

This will fail every login attempt as both strings cannot be valid simultaneously.

Reply:
Thanks for the suggestion.
It should work because we use `!=` in the string comparison for the "failing both" logic.
But I agree to change it to an OR for the "not matching any" logic to clearly match the documentation.